integration-tests/utils/zendesk/createZendeskTicket.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+import { authoriseAs } from './authoriseAs'
+import {
+  ZENDESK_BASE_URL,
+  ZENDESK_END_USER_EMAIL,
+  ZENDESK_REQUESTS_ENDPOINT
+} from '../../constants/zendeskParameters'
+import { ZendeskRequestData } from '../../types/zendeskRequestData'
+
+export const createZendeskTicket = async (requestData: ZendeskRequestData) => {
+  try {
+    const response = await axios({
+      url: `${ZENDESK_BASE_URL}${ZENDESK_REQUESTS_ENDPOINT}`,
+      method: 'POST',
+      headers: {
+        Authorization: authoriseAs(ZENDESK_END_USER_EMAIL),
+        'Content-Type': 'application/json'
+      },
+      data: requestData
+    })
+    return response.data.request.id.toString()
```

Comment:
Can we log the ticket id? It would help for checking in Zendesk when debugging.

Reply:
Done